src/Foundation/ErrorHandling/Middleware/ExecuteErrorToFrontend.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Flarum\Foundation\ErrorHandling\Middleware;
+
+use Flarum\Http\RouteHandlerFactory;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+class ExecuteErrorToFrontend implements MiddlewareInterface
```

Comment:
Is this too verbose? `ExecuteError`?

Reply:
Verbosity === self-documentation in my eyes! )